integrations/elasticsearch/elasticsearch-events/README.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Elasticsearch Events integration provides a pipeline for sending metrics and events from Sensu to Elasticsearch.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `elasticsearch` [pipeline]
+* `elasticsearch-events` [handler]
+* `ELASTICSEARCH_USERNAME` [secret]
```

Comment:
```suggestion
* `elasticsearch_username ` [secret]
```

Looks like the Sensu resource name is lowercase. 

See: https://github.com/sensu/catalog/pull/151/files#diff-21f1febc19271950dfb84121b479eb5d750976c62084c9f24738d1a99c6974d2R48